### DISCUSSION TITLE
Automatic Reverse Alias

### DISCUSSION THREAD

**Discussion by tangiblehologram** [simple-login/app#1156] [Feature Requests]
I would very much like to have automatic reverse alias capabilities as part of the Proton and SimpleLogin partnership.

This would require configuration settings such as:
- automatic reverse aliasing on/off

It would require an adjustment of the Proton Mail interface
- Add aliases/create new new alias to send from dropdown

**Comment by nukeseo** [comment]
If this means enabling Proton Mail aliases to be selected as forwarding addresses in SimpleLogin and vice versa, I would also like it.  Enabling a Proton Mail alias to receive email forwarded from SimpleLogin enables email filtering and default reply signatures.  Very useful!
